src/components/SignInOrJoinFree.js

```diff
@@ -56,6 +57,25 @@ class SignInOrJoinFree extends React.Component {
     return encodeURIComponent(this.props.redirect || window.location.pathname || '/');
   }
 
+  signInBlockstack = async data => {
+    if (this.state.submitting) {
+      return false;
+    }
+    this.setState({ submitting: true });
+
+    const next = data.redirect || '/';
+    let redirectUrl;
+    if (data) {
+      const organizationData = pick(data, ['orgName', 'githubHandle', 'twitterHandle', 'website']);
```

Comment:
If we remove the `createProfile` form, can that be removed?

Reply:
Removed in https://github.com/opencollective/opencollective-frontend/pull/1980